otx/cli/builder/builder.py

```diff
@@ -0,0 +1,276 @@
+"""Builder Class for training template."""
+
+# Copyright (C) 2022 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import inspect
+import os
+import shutil
+
+import mmcv
+import torch
+from mmcv.utils import Registry, build_from_cfg
+from mpa.utils.config_utils import MPAConfig
+
+from otx.cli.registry import Registry as OTXRegistry
+from otx.cli.utils.importing import get_backbone_registry
+
+DEFAULT_MODEL_TEMPLATE_ID = {
+    "classification": "Custom_Image_Classification_EfficinetNet-B0",
+    "detection": "Custom_Object_Detection_Gen3_ATSS",
+    "instance_segmentation": "Custom_Counting_Instance_Segmentation_MaskRCNN_ResNet50",
+    "segmentation": "Custom_Semantic_Segmentation_Lite-HRNet-18-mod2_OCR",
+}
+
+
+# pylint: disable=too-many-locals, too-many-statements, too-many-branches
+
+
+def get_backbone_out_channels(backbone):
+    """Get output channels of backbone using fake data."""
+    out_channels = []
+    input_size = backbone.input_size if hasattr(backbone, "input_size") else 64
```

Comment:
I think 64 is too small for default input resolution.
Did you check this value for multiple models?

Reply:
I tested ATSS and Mask-RCNN by fixing the current value (64). Do you think this will have a big impact on other models?

Reply:
This function just calculates the output channel, so I think the input resolution doesn't matter.